npm/private/npm_translate_lock_generate.bzl

```diff
@@ -566,6 +567,57 @@ or disable this check by setting `verify_node_modules_ignored = None` in `npm_tr
             )
             fail(msg)
 
+################################################################################
+# https://github.com/bazelbuild/bazel/issues/3901#issuecomment-421318536
+def _directory_path(repository_ctx, label):
```

Comment:
mind moving these into `npm_translate_lock.bzl` or perhaps a new `npm_translate_lock_helpers.bzl`. The `helpers` struct in this one is an artifact of the big refactor since I wanted to minimize the diff. I plan to move the functions that are not starlark generation specific out of this file.